docs/portal.rst

```diff
@@ -0,0 +1,208 @@
+=====================
+The Management Portal
+=====================
+
+#. Prerequisites_
+
+#. `Ready, Set, ASAP! (It's quick I promise)`_
+
+#. `The TL;DR section`_
+
+    - `Creating and testing React components`_
+    - Build_
+    - Preview_
+    - `Add images`_
+    - `Add Asset folder`_
+    - `Run Flask daemon`_
+    - `The Flask daemon`_
+    - `I npm ejected`_
+    - `TailwindCSS`_
+    - `Custom CSS styles`_
+    - `Apply custom CSS styles`_
+    - Troubleshooting_
+
+Prerequisites
+-------------
+You must have Pattoo already installed and the database set up.
+Please see installation_ documentation for more info.
+
+.. _installation: https://github.com/PalisadoesFoundation/pattoo/blob/master/docs/installation.rst#basic-installation
+
+Ready, Set, ASAP! (It's quick I promise)
+----------------------------------------
+
+Ready
```

Comment:
Not acceptable. The installation cannot be separate from the main installation. The user must not install twice. This needs to be integrated.